<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borde</class>
 <widget class="QWidget" name="borde">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>borde</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>风扇打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>蜂鸣器打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>马达打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>湿度：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>打开灯光</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>获取温湿度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>201</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>打开门锁</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
